src/edit/contributorFields.js

```diff
@@ -74,6 +68,35 @@ const ContributorFields = ({
               name: 'contributorTypeText',
               component: TextField,
             },
+            {
+              name: 'primary',
+              label: intl.formatMessage({ id: 'ui-inventory.primary' }),
+              component: ({ label, meta, input, fields }) => { /* eslint-disable-line react/prop-types */
```

Comment:
What do you think about moving this field to a component?

Reply:
I think that's a good idea. I'll move it into a new component 👍🏻